mappings/net/minecraft/world/gen/OreVeinSampler.mapping

```diff
@@ -0,0 +1,21 @@
+CLASS net/minecraft/class_6955 net/minecraft/world/gen/OreVeinSampler
+	FIELD field_36627 RAW_BLOCK_CHANCE F
+	METHOD method_40547 (Lnet/minecraft/class_6910;Lnet/minecraft/class_2680;Lnet/minecraft/class_6574;Lnet/minecraft/class_6910;Lnet/minecraft/class_6910;Lnet/minecraft/class_6910$class_6912;)Lnet/minecraft/class_2680;
+		ARG 5 startPos
```

Comment:
Ore veins don't have a start position as far as I know. Instead, the chunk generator calls this lambda for every position not covered by the aquifer sampler. Notice that it returns `null` to use the fallback state/the chunk generator's default block state. Just `pos` or `noisePos`?